GLMakie/src/screen.jl

```diff
@@ -388,19 +405,28 @@ end
 function Screen(;
         resolution::Union{Nothing, Tuple{Int, Int}} = nothing,
         start_renderloop = true,
+        window = nothing,
         screen_config...
     )
     # Screen config is managed by the current active theme, so managed by Makie
     config = Makie.merge_screen_config(ScreenConfig, Dict{Symbol, Any}(screen_config))
-    screen = screen_from_pool(config.debugging)
+    screen = screen_from_pool(config.debugging; window)
     apply_config!(screen, config; start_renderloop=start_renderloop)
     if !isnothing(resolution)
         resize!(screen, resolution...)
     end
     return screen
 end
 
-set_screen_visibility!(screen::Screen, visible::Bool) = set_screen_visibility!(screen.glscreen, visible)
+function set_screen_visibility!(screen::Screen, visible::Bool)
```

Comment:
Should these error instead of returning?
I Think for Makie to work correctly, these will need to be overloaded for `Screen{T}`, so they should not end up here for screens not owned by makie, no?
Same for the other functions that noop, when not owning!

Reply:
Hmm, I originally thought that it flat-out shouldn't be allowed for Makie to touch the window but I guess that should be left up to the integration. Changed it to an error in e2edca596.